src/platform/sourcemapped-stacktrace-node.js

```diff
@@ -6,4 +6,4 @@
 // http://polymer.github.io/PATENTS.txt
 
 // This is only relevant in the web devtools.
-export const mapStackTrace = () => {};
+export const mapStackTrace = (x, f) => f([x]);
```

Comment:
I think this deserves a comment to explain why we have something called mapStackTrace (especially as it = Data.Function.(&) rather than something doing mapping).

Reply:
done